packages/aragon-wrapper/src/index.js

```diff
@@ -111,9 +112,7 @@ export default class Aragon {
    * @param {?Array<string>} [accounts=null] An optional array of accounts that the user controls
    * @return {Promise<void>}
    */
-  async initAccounts (accounts) {
-    this.accounts = new ReplaySubject(1)
```

Comment:
We don't return the observable, we convert it into a promise, so we only use `ReplaySubject` internally:
```
  getAccounts () {
    return this.accounts
      .take(1)
      .toPromise()
  }
```
Anyway, I reverted it back since it's a silly change and not important to this PR.